src/screens/staking/lib/staking_sdk/utils/coins.ts

```diff
@@ -24,9 +28,14 @@ type DenomToNormalise = (typeof networkToUnnormalisedDenom)[StakingNetworkId];
 const uExp = 6;
 const pExp = 12;
 const aExp = 18;
+const exp0 = 0;
 
 const denomMap: Record<DenomToNormalise, [CoinDenom, number]> = {
   ADYDX: [CoinDenom.DYDX, aExp],
+  ADYM: [CoinDenom.DYM, aExp],
+  AEVMOS: [CoinDenom.EVMOS, aExp],
+  AISLM: [CoinDenom.ISLM, aExp],
+  inj: [CoinDenom.INJ, exp0], // This is handled when normalising the coin
```

Comment:
Any reason why `inj` is in lowercase here?

Reply:
I improve the comment, this is a special case where `inj` denom doesn't have the same meaning as `INJ`, so it is handled separately.